### PR TITLE
Fixed issue #1195

### PR DESCRIPTION
**Pull request by phecho**

### CODE REVIEW

File: resources/views/dashboard/partials/head.blade.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <link rel="apple-touch-icon" sizes="144x144" href="/img/apple-touch-icon-144x144.png">
     <link rel="apple-touch-icon" sizes="152x152" href="/img/apple-touch-icon-152x152.png">
 
-    <title>@yield('pageTitle', $app_name.' | Cachet')</title>
+    <title>{{ $page_title or  $app_name.' | Cachet' }}</title>
```
Comment (GrahamCampbell) [inline resources/views/dashboard/partials/head.blade.php]: too many spaces after `or`

Comment (GrahamCampbell) [inline resources/views/dashboard/partials/head.blade.php]: also please, please just write `$page_title ?: $app_name`

Comment (phecho) [inline resources/views/dashboard/partials/head.blade.php]: @GrahamCampbell 
no. they are different.

Comment (GrahamCampbell) [inline resources/views/dashboard/partials/head.blade.php]: `$page_title ?: $app_name.' | Cachet'` then

Comment (phecho) [inline resources/views/dashboard/partials/head.blade.php]: @GrahamCampbell 
```html
<title>{{ $page_title or  $app_name.' | Cachet' }}</title>
```
equal
```html
<title>{{ isset($page_title) && $page_title ?:  $app_name.' | Cachet' }}</title>
```

Comment (GrahamCampbell) [inline resources/views/dashboard/partials/head.blade.php]: Right, ok. Leave it then. ;)

**Pull request merged**